phoenix-core/src/main/java/org/apache/phoenix/query/QueryServices.java

```diff
@@ -322,6 +322,12 @@
 
 	public static final String SYSTEM_CATALOG_SPLITTABLE = "phoenix.system.catalog.splittable";
 
+    // The parameters defined for handling task stored in table SYSTEM.TASK
+	// The time interval between periodic scans of table SYSTEM.TASK
+    public static final String TASK_HANDLING_INTERVAL_ATTRIB = "phoenix.task.handling.interval";
```

Comment:
One small nit: Can we add timeunits in the property name?

Reply:
Will change it to TASK_HANDLING_INTERVAL_MS_ATTRIB